www/_posts/2022-07-12-cordova-android-release-11.0.0.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 11.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 11.0.0`! This is one of Cordova's supported platforms for building Android applications.
+
+* [cordova-android@11.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@11.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add android@11.0.0
+```
+
+* **Android 12 SplashScreen API Integration**
+
+    As of Android 12, all Android 12 or higher devices display the new app launch animation. Google has applied this requirement to bring a standard design to all app launch screens.
+
+    Due to this requirement, users have seen and reported the display of multiple SplashScreens during the app launch. Since the new SplashScreen API can not be disabled, the old SplashScreen plugin is deprecated for Cordova-Android 11+.
+
+    We have integrated the Android 12 SplashScreen API including the compatibility library into the core of the Cordova-Android platform to provide support for Android API 22+.
+
+    Only some of the exisiting SplashScreen configurations from the old plugin are still supported. For example: `AutoHideSplashScreen`, `SplashScreenDelay`, `FadeSplashScreen`, & `FadeSplashScreenDuration`.
+
+    With the integration of SplashScreen API 12, the following settings have been introduced:
+
+    * `AndroidWindowSplashScreenAnimatedIcon`: Contains the path of the resource image of the SplashScreen Icon. Either an `XML Android Drawable` or `PNG`
+    * `AndroidWindowSplashScreenBackground`: Contains the hex value of the SplashScreen background color. Default is white (#FFFFFF)
+    * `AndroidWindowSplashScreenIconBackgroundColor`: Contains the hex value of the SplashScreen Icon's background color. Default is not defined.
+
+    Also, there were also small behavior changes:
+
+    * `SplashScreenDelay` with `AutoHideSplashScreen` enabled will automatically hide the SplashScreen after the `onPageFinished` is triggered. The original behavior can be restored by setting the setting the `SplashScreenDelay` preference to `3000`, if preferred.
+    * `FadeSplashScreen` only controls the fade out.
+
+    For more information, please refer to the [PR](https://github.com/apache/cordova-android/pull/1441) and [Cordova Docs](https://cordova.apache.org/docs/en/latest/core/features/splashscreen/index.html).
```

Comment:
```suggestion
    As of Android 12, all Android 12 or higher devices display the new app launch animation. Google has applied this requirement to bring a standard design to all app launch screens.

    Due to this requirement, users have seen and reported the display of multiple SplashScreens during the app launch. Since the new SplashScreen API can not be disabled, the old SplashScreen plugin is deprecated for Cordova-Android 11+.

    We have integrated the Android 12 SplashScreen API including the compatibility library into the core of the Cordova-Android platform to provide support for Android API 22+.

    For more information, please refer to the [PR](https://github.com/apache/cordova-android/pull/1441) and [Cordova Docs](https://cordova.apache.org/docs/en/latest/core/features/splashscreen/index.html).
```